examples/gno.land/p/gov/dao/dao.gno

```diff
@@ -0,0 +1,14 @@
+package dao
+
+// DAO defines the DAO abstraction
+type DAO interface {
+	// Propose adds a new proposal to the executor-based GOVDAO.
+	// Returns the generated proposal ID
+	Propose(comment string, executor Executor) (uint64, error)
```

Comment:
```suggestion
	Propose(prop Proposal) (uint64, error)
```
We should create a `Proposal` interface that, at a minimum, returns a `GetComment()` method and a `GetExecutor()` method. This will allow the proposal to contain more elements in the future while ensuring it always has these core components.

Reply:
Oh, the interface already exists, great; plase just add GetComment, or rename so Comment matches with Description.

And includes an `Execute() error` handler.

Reply:
The `Proposal` is something that wraps information you cannot know at "propose time" (like the voting info, what is the author, status...)

I've defined a new type `ProposalRequest`, and added the methods requested:

[74948fc](https://github.com/gnolang/gno/pull/2581/commits/74948fcdb13b276830857359bc83a7c20d6085bf)